resources/dev/config.yml

```diff
@@ -0,0 +1,48 @@
+#
+# Configuration file for Docker registry
+# https://distribution.github.io/distribution/about/deploying/
+#
+# Before using, notifications/endpoints/headers/Authorization must be filled in
+#
+version: 0.1
+log:
+    level: info
+    formatter: text
+    fields:
+        service: registry
+http:
+    addr: :443
```

Comment:
I think we also need the value for 'secret' here (I saw it on the running instance, as I recall it was necessary with multiple instances).

Reply:
No, the config' you saw previously had a mistake:  `secret` was a parameter used for a different kind of authentication.